gnarly.rb

```diff
@@ -197,7 +207,6 @@ def setup_analysis
 
 def setup_linting
   copy_file "templates/.rubocop.yml", ".rubocop.yml"
-  copy_file "templates/.scss-lint.yml", ".scss-lint.yml"
```

Comment:
scss should not be linted by ruby, and this library looked for the code in the wrong places. 

I did not go through the process of manually including a JS based linter here, but that should come soon.